src/components/SelectableMainTable.tsx

```diff
@@ -195,7 +197,7 @@ const SelectableMainTable: FC<Props> = ({
 
     const className = classnames(row.className, {
       "selected-row": isRowSelected,
-      "processing-row": isRowProcessing,
+      "processing-row": isRowDisabled,
```

Comment:
Should rename the css class as well.
```suggestion
      "disabled-row": isRowDisabled,
```